clock_exercise/TimerButton.qml

```diff
@@ -0,0 +1,43 @@
+import QtQuick 2.15
+
+Item {
+    id: root
+
+    property alias activeImageSource: downButton.source
+    property alias hoverImageSource: hoverImage.source
+
+    signal clicked
+
+    width: 145
+    height: 145
+
+    Image {
+        id: downButton
+
+        opacity: mouseArea.containsMouse? 0: 1
+        anchors.fill: parent
+
+        Behavior on opacity {
+            NumberAnimation { duration: 500 }
+        }
+
+        MouseArea {
```

Comment:
La MouseArea la terrei come elemento root, è sempre la cosa più comoda quando si hanno queste componenti in stile `Button`.

Reply:
Anche perché così chi usa questo componente ha a disposizione tutte le property di `MouseArea` per personalizzare il comportamento. Es. se volessi un long press? Posso usare `onPressAndHold` e `pressAndHoldInterval` per fare le mie personalizzazioni